app/actions/local/category.ts

```diff
@@ -30,7 +30,7 @@ export const deleteCategory = async (serverUrl: string, categoryId: string) => {
     }
 };
 
-export const storeCategories = async (serverUrl: string, categories: CategoryWithChannels[], prune = false, prepareRecordsOnly = false) => {
```

Comment:
This was a fruitless effort to get rid of an anonymous function in the logs. I wasn't able to find out where this anonymous function comes from, but is somewhere between the action call (in particular the entry action, but probably any action) and the batchRecords call.

Nevertheless, since the work was done, I left it in place.